jekyll/_cci2/github-apps-integration.adoc

```diff
@@ -0,0 +1,255 @@
+- At this time, the **Free and Open Source** setting is not currently supported, but there are plans to make this available in the future.
+- At this time, auto-cancel redundant workflows is not supported. Refer to the xref:skip-build#auto-cancelling[Auto cancelling] section of the `skip` or `cancel` jobs and workflows page for more details.
+
+[#project-settings-ssh-keys]
+=== GitHub project SSH keys
+
+When creating a project in CircleCI, you will create and add SSH keys, which are used to check out code from your repo. Each configuration you create generates a new SSH key to access the code in the repo associated with that configuration. At this time, only **Additional SSH Keys** are applicable to GitHub App integrations.
+
+[#create-ssh-key]
+==== Create an SSH key
+
+You will be guided through the SSH key generation processing in the CircleCI web app. The steps are as follows:
+
+. Create an SSH key-pair by using the following command. When prompted to enter a passphrase, do **not** enter one:
++
+```shell
+  ssh-keygen -t ed25519 -C "your_email@example.com"
+```
+
+. Go to your project on link:https://gitlab.com/[GitLab] and navigate to **Settings > Repository**, and expand the **Deploy keys** section. Enter a title in the "Title" field, then copy and paste the public key you created in step 1. Check **Grant write permissions to this key** box, then click **Add key**.
+
+. Go to your project settings in the CircleCI app, select **SSH Keys**, and **Add SSH key**. In the "Hostname" field, enter `gitlab.com` and add the private key you created in step 1. Then click **Add SSH Key**.
+
+When you push to your GitLab repository from a job, CircleCI will use the SSH key you added.
+
+For more information on SSH keys, please visit the xref:add-ssh-key#[Adding an SSH key to CircleCI] page.
+
+[#organization-settings]
+== Organization settings
+
+For GitHub Apps integrations, organizations and users are managed independently from your VCS. Organizations and users are considered CircleCI organizations and users, with their own roles and permissions that do not rely on those defined in your VCS.
+
+To manage settings at the organization level, click btn:[Organization Settings] in the CircleCI web app sidebar.
+
+[#organization-settings-people]
+=== People
+
+Add or remove users, and manage user roles for the organization as well as user invites. See the xref:roles-and-permissions-overview#[Roles and permissions overview] page for full details.
+
+[#roles-and-permissions]
+== Roles and permissions
+
+CircleCI users have different abilities depending on assigned roles in a particular organization. For a detailed list of CircleCI org and project roles and associated permissions, see the xref:roles-and-permissions-overview#[Roles and permissions] page.
+
+[#user-settings]
+== User settings
+
+[#user-account-integrations]
+=== Account integrations
```

Comment:
@Andrewwelton is this section relevant? Can existing GitLab users connect to GtiHub Apps this way for example?
![Screenshot 2023-08-03 at 15 30 59](https://github.com/circleci/circleci-docs/assets/24589403/c3397325-6fe7-4d7a-accf-ed9a524e15d0)

Reply:
Nope, this is to connect to the legacy GitHub integration